### ISSUE TITLE
WKWebView refresh

### ISSUE THREAD

**Issue by vijaydogra** [WildDylan/WKWebViewWithURLProtocol#8]
After putting in these files in my project, WKWebView is refreshing on every close of dialog or modal windows. When I comment out the registerSchemeClass call, it works fine. So just wanted to know if someone else had faced this issue, and if yes, how was it solved?

**Comment by wilddylan** [comment]
> WKWebView is refreshing on every close of dialog or modal windows; 

can you provide the full html code(js,css,html) for open/hide dialog and modal window and callback code ?

**Comment by julian-weinert** [comment]
Yeah, I can confirm. Unfortunately your way isn't supported anymore. See this comment for details https://stackoverflow.com/a/54821268/1041122

```
[<WKWebView 0x600003e0a700> valueForUndefinedKey:]: this class is not key value coding-compliant for the key browsingContextController.
```

Unfortunately that throws a big stone in a new project of mine. I'm already considering implementing the legacy `WebView` but honestly I don't want to implement something I may not be able to replace...
